examples/hello-world-lbaware-server/src/main/java/com/rtbhouse/grpc/lbexamples/HelloWorldLBServer.java

```diff
@@ -0,0 +1,68 @@
+package com.rtbhouse.grpc.lbexamples;
+
+import com.rtbhouse.grpc.loadbalancer.BasicLoadbalancerAwareGrpcServer;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HelloWorldLBServer extends BasicLoadbalancerAwareGrpcServer {
```

Comment:
Ten tak jak wszystkie inne przykłady rozszerzają GrpcServer? Czemu? Czy GrpcServer  musi być rozszerzany? nie może być użyty jako komponent? Jeśli może to zdrowsze przykłady będą które nie dziedziczą po GrpcServer.

Reply:
Poprawione - tak jak dyskutowaliśmy faktycznie tak jest zdrowiej.